### PR TITLE
Fix/as 4000 data not removed

### PR DESCRIPTION
**Pull request by shilpabasu22**

### CODE REVIEW

File: e2e-tests/cypress/support/db-queries/select-case-reference-from-db.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 export const selectCaseReferenceFromDb = (status) =>{
     if(status=== 'Received'){
-        cy.sqlServer("select caseReference from [QuestionnaireData] where questionnaireStatus='RECEIVED'").then( (res)=>{
+        cy.sqlServer("select * from [QuestionnaireData] where questionnaireStatus='RECEIVED'").then( (res)=>{
+            console.log(res);
```
Comment (demzzydev) [inline e2e-tests/cypress/support/db-queries/select-case-reference-from-db.js:4]: Do we need to log this?

**Review by demzzydev — APPROVED** [review]
LGTM

**Review by karunak21 — APPROVED** [review]
all good

**Pull request merged**